tests/integration/Subscribers/InactiveSubscribersControllerTest.php

```diff
@@ -188,6 +208,24 @@ function testItDoesNotActivateOldSubscribersWithUnopenedEmail() {
     expect($subscriber->status)->equals(Subscriber::STATUS_INACTIVE);
   }
 
+  function testItActivatesSubscribersWhenMP2MigrationHappenedWithinInterval() {
```

Comment:
This test is very hard to understand. We should probably stop using numbers and use constants or variables instead.